src/styles/blocks/_header.scss

```diff
@@ -0,0 +1,119 @@
+.header {
+
+  &__top {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    height: 76px;
+
+    @include onTablet {
+      margin-bottom: 120px;
+    }
+
+    @include onDesktop {
+      margin-bottom: 160px;
+    }
+
+    @include onTablet {
+      &-desktop {
+        width: 300px;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+      }
+    }
+  }
+
+  &__logo {
+    @extend %logo;
+  }
+
+  &__specs {
+    @extend %link;
+
+    color: #4f4f4f;
+    font-weight: 300;
+    font-size: 14px;
+    line-height: 17px;
+  }
+
+  &__language {
+    display: flex;
+  }
+
+  &__button {
+    @extend %flex-center;
+    @extend %link;
+    @extend %button;
+
+    height: 48px;
+    width: 102px;
+    background-color: $dark-color;
+    transition: 0.3s;
+
+    &:hover {
+      background: #131313;
+    }
+
+    &:disabled {
+      background: #d0d0d0;
+    }
+  }
+
+  &__inner {
+    @include onTablet {
+      display: grid;
+      grid-template-columns: repeat(8, 1fr);
+      grid-template-rows: 1fr;
+      grid-auto-flow: dense;
+    }
```

Comment:
better create separate file for grid, as you can see in [Miami grid - video](https://mate.academy/learn/css-advanced/css-advanced-landing#/video)